presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeIdenticalWindows.java

```diff
@@ -249,15 +249,14 @@ public void testNotMergeDifferentFrames()
         assertUnitPlan(sql,
                 anyTree(
                         window(ImmutableList.of(
-                                functionCall("avg", windowD, false, "quantity")),
-                                window(ImmutableList.of(
-                                        functionCall("sum", windowC, false, "discount"),
-                                        functionCall("sum", windowC, false, "quantity")),
-                                        any()))));
+                                functionCall("avg", windowD, false, "quantity"),
```

Comment:
Given my comment above, the changes to these tests should probably go in the commit that introduces the optimization.